packages/ffe-searchable-dropdown-react/src/SearchableDropdown.md

```diff
@@ -1,47 +1,54 @@
-The default:
+En komponent for å kunne søke igjennom en lenger liste med valg. Listen med valg har støtte for å vise flere atributter ved et element, for eksempel bedriftnavn og organisasjonsnummer.
```

Comment:
attributter*

Reply:
Noticed the same typo a few other places too. Perhaps you can spell it both ways? 🤷‍♂️

Reply:
Darnit, Google-as-a-spellchecker failed me :(